src/commands/synthetics/interfaces.ts

```diff
@@ -262,18 +277,24 @@ export interface LocalTestDefinition {
   }
   /** Can be used to link to an existing remote test. */
   public_id?: string
-  subtype: string
-  tags: string[]
+  subtype?: string
+  steps?: TestStepWithUnsupportedFields[] // From browser schema
```

Comment:
Here I think this should do the job, and you can remove `TestStep` which is redundant with `Step` and also remove `TestStepWithUnsupportedFields`.
You can then use `ServerTest["steps"]` to type `STEP_FIELDS_TRIM`
```suggestion
  steps?: {params: any, position: number, public_id: string}[] // From browser schema
```